recipes/discretisedfield/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "0.7.2" %}
+{% set name = "discretisedfield" %}
+{% set sha256 = "e52c78f72f181edd328872e541564e0a4bee17d73d575111360574032fdf992c" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py27]
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy >=1.11
```

Comment:
In the build step, the script given above (`pip install --no-deps .`) runs. Which means the files are installed and also `pyc` files are created. Creating pyc files doesn't care about imports.